velodyne_driver/src/lib/input.cc

```diff
@@ -114,7 +114,7 @@ namespace velodyne_driver
   /** @brief Get one velodyne packet. */
   int InputSocket::getPacket(velodyne_msgs::VelodynePacket *pkt, const double time_offset)
   {
-    double time1 = ros::Time::now().toSec();
+    const ros::Time time1 = ros::Time::now();
```

Comment:
`now` would be a great name for this variable

Reply:
fixed.